presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/file/FileHiveMetastore.java

```diff
@@ -411,18 +416,11 @@ public synchronized void updatePartitionStatistics(HiveIdentity identity, Table
     @Override
     public synchronized List<String> getAllTables(String databaseName)
     {
-        requireNonNull(databaseName, "databaseName is null");
-
-        Optional<Database> database = getDatabase(databaseName);
-        if (database.isEmpty()) {
-            return ImmutableList.of();
-        }
-
-        Path databaseMetadataDirectory = getDatabaseMetadataDirectory(databaseName);
-        List<String> tables = getChildSchemaDirectories(databaseMetadataDirectory).stream()
-                .map(Path::getName)
+        return listAllTables(databaseName).stream()
```

Comment:
Do we even support File Hive metastore for Delta Lake?

Reply:
Not sure what this question is about?